vb-be/src/modules/room/room.controller.ts

```diff
@@ -53,7 +53,7 @@ export class RoomController {
         throw new ErrorHandler(HttpStatus.InternalError, ErrorText.Generic);
       }
     } catch (err) {
-      return res.status(err.statusCode).send(err.message);
+      return res.status(err.statusCode || err.status).send(err.message);
```

Comment:
Got an internal error about invalid status code - something must have occurred that wasn't due to one of our error handlers, I think this may resolve it.